xml/security_ldap.xml

```diff
@@ -398,266 +399,195 @@ objectclass (2.16.840.1.113730.3.2.333 NAME 'nsPerson' <co xml:id="co-ldap-schem
+suffix = <replaceable>dc=test1,dc=com</replaceable></screen>
+
       <calloutlist>
+       <callout arearefs="co-ldap-389-ds-config_version">
+        <para>
+         This line is required, indicating that this is a version 2 setup .inf
+         file.
+        </para>
+       </callout>
        <callout arearefs="co-ldap-389-ds-rootpasswd">
         <para>
-         Set the <varname>root_password</varname> to the password for
-         the directory server &rootuser; user. The password is used for LDAP
-         server administration only.
+         Create a <varname>root_password</varname> for
+         the directory server &rootuser; user, <literal>dirsrv</literal>. 
+         The <literal>dirsrv</literal> user has no login or shell privileges, 
+         and is used for LDAP server administration only.
+        </para>
+       </callout>
+       <callout arearefs="co-ldap-389-ds-cert">
+        <para>
+         Create self-signed server certificates in 
+         <filename>/etc/dirsrv</filename>.
         </para>
        </callout>
+       <callout arearefs="co-ldap-389-ds-sample-entries">
+        <para>
+         Populate the new instance with sample user and group entries.
+        </para>
+       </callout>       
       </calloutlist>
      </example>
    </step>
    <step>
     <para>
      To create the &ds389; instance from <xref linkend="ex-ldap-389-ds-inf"
-     xrefstyle="select:label"/>, run:
+     xrefstyle="select:label"/>, run the following command:
     </para>
-    <screen>&prompt.sudo;dscreate from-file /tmp/instance.inf</screen>
+    <screen>&prompt.sudo;dscreate -v from-file 389ds-test.inf | tee 389ds-test-output.txt</screen>
     <para>
-     This creates a working LDAP server.
+     This shows all activity during the instance creation, stores all the messages in
+     <filename>389ds-test-output.txt</filename>, and creates a working 
+     LDAP server in about a minute. The verbose output contains a lot of useful
+     information. If you don't want to save it then delete the 
+     <literal>| tee 389ds-test-output.txt</literal> portion of the command.
     </para>
    </step>
    <step>
     <para>
-     If <command>dscreate</command> should fail, the messages will tell you why.
-     For more details, repeat the command with the <option>-v</option> option:
+     If the <command>dscreate</command> command should fail, the messages will 
+     tell you why. After correcting any issues, re-run the command.    
     </para>
-    <screen>&prompt.sudo;dscreate -v from-file /tmp/instance.inf</screen>
    </step>
    <step>
     <para>
-     Check the status of the server with:
+     A successful installation reports "<literal>Completed installation for 
+     389ds-test</literal>". Check the status of your new server:
     </para>
-    <screen>&prompt.sudo;dsctl localhost status
-instance 'Localhost' is running</screen>
+    <screen>&prompt.sudo;dsctl 389ds-test status
+Instance "389ds-test" is running</screen>
```

Comment:
Please file a bug, that this needs more 🎉 emoji.

Reply:
It does! It totally deserves a Bacchanalia emoji.